main.cpp

```diff
@@ -49,8 +50,119 @@
 
  Wait for my code review.
  */
-
 #include <iostream>
+
+struct FloatType
+{
+    float add( float lhs, float rhs );
+    float subtract( float lhs, float rhs );
+    float multiply( float lhs, float rhs );
+    float divide( float lhs, float rhs );
+};
+
+float FloatType::add( float lhs, float rhs )
+{
+    return lhs + rhs;
+}
+
+float FloatType::subtract( float lhs, float rhs )
+{
+    return lhs - rhs;
+}
+
+float FloatType::multiply( float lhs, float rhs )
+{
+    return lhs * rhs;
+}
+
+float FloatType::divide( float lhs, float rhs )
+{
+    if ( rhs == 0.0f )
+    {
+        std::cout << "\nwarning, floating point division by zero returns 'inf' !\n";
+        return lhs / rhs;
+    }
+    else
+    {
+        return lhs / rhs;
+    }
+}
```

Comment:
You wrote `return lhs / rhs;` twice.  Don't Repeat Yourself.  D.R.Y.
Move it outside of the `if/else` block
- [ ] revise